src/test/java/org/folio/service/transformationfields/TransformationFieldsServiceUnitTest.java

```diff
@@ -96,6 +96,8 @@ void getFieldNamesShouldReturnValidFields(VertxTestContext context) {
         TransformationFieldCollection transformationFieldCollection = ar.result();
         transformationFieldCollection.getTransformationFields()
           .forEach(transformationField -> checkIfActualFieldEqualToExpected(expectedFields.get(transformationField.getFieldId()), transformationField));
+
+       System.out.println(transformationFieldCollection);
```

Comment:
I think this sout shout be removed)

Reply:
yup removed

Reply:
I do not see, please commit changes

Reply:
I think you may have to refresh, here's the commit
https://github.com/folio-org/mod-data-export/pull/140/commits/bebd0301bf9c151a356c8352ab8414421bc80b50